code/game/verbs/who.dm

```diff
@@ -3,46 +3,44 @@
 	set category = "OOC"
 
 	var/list/lines = list()
-	var/list/temp = list()
+	var/list/ckeys = list()
 
 	for(var/client/C in GLOB.clients)
 		if(C.holder && C.holder.big_brother) // BB doesn't show up at all
 			continue
 
 		if(C.holder && C.holder.fakekey)
-			temp += C.holder.fakekey
+			ckeys.Add(C.holder.fakekey)
 		else
-			temp += C.key
+			ckeys.Add(C.key)
```

Comment:
iirc += is faster

(to be clear I hate that that is the fact)

Reply:
We standardised the other way I think? Cc @ParadiseSS13/headcoders